doc/user/logging.md

```diff
@@ -60,6 +60,53 @@ $ go run main.go --zap-level=1
 {"level":"info","ts":1559866310.065119,"logger":"scoped","msg":"Printing at INFO level"}
 {"level":"debug","ts":1559866310.065123,"logger":"scoped","msg":"Printing at DEBUG level"}
 ```
+## Custom zap logger
+
+In order to use a custom zap logger, [`zap`][controller_runtime_zap] from controller-runtime can be utilized to wrap it in a logr implementation.
+
+Below is an example illustrating the use of [`zap-logfmt`][logfmt_repo] in logging.
+
+### Example
+
+```Go
+package main
+
+import (
+	"github.com/go-logr/zapr"
+	zaplogfmt "github.com/sykesm/zap-logfmt"
```

Comment:
Why we need the `zaplogfmt "github.com/sykesm/zap-logfmt"`? Should we not use what the controller-runtime offers?